README.md

```diff
@@ -1,16 +1,248 @@
-# Title
+# Methods in Ruby
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Describe routines and procedures for our code using methods
+- Define a method
+- Recognize how to call methods
 
 ## Introduction
 
-## SWBAT 1
+A method in Ruby is a _block_ of code that returns a value. With methods, we can
```

Comment:
I'd suggest swapping the first 2 paras with this (or something close):

{what is a method}
A method in Ruby is a group of expressions or statements that work together to do something. With methods, we can organize our code into chunks of reusable functionality.

{ be clear about return values}
Most methods return a value. That means they're behaving a like an _expression_. Other times a method might merely do work (like printing stuff to the screen) and the return value is not so important.